### ISSUE TITLE
Task - species described in vs. currently in

### ISSUE THREAD

**Issue by mjy** [SpeciesFileGroup/taxonworks#740]
As a user I want to see a report  (likely a map) that concisely reflects the following:

* The country of orgin of a valid species (as referenced by type material)
* The current known distribution of that species.

Bonus:

* Animate when (year) each record appears, from oldest year to newest year

**Comment by proceps** [comment]
is it coveredy by browe OTU?

**Comment by mjy** [comment]
Yes, it will be when we highlight type specimens differently than others on the specimen map. We likley need a custom geo-json response to optimize this plot.
